typescript/infra/config/environments/testnet2/helloworld.ts

```diff
@@ -14,7 +14,15 @@ export const abacus: HelloWorldConfig<TestnetChains> = {
       repo: 'gcr.io/abacus-labs-dev/hyperlane-monorepo',
       tag: 'sha-fcc80c8',
     },
-    chainsToSkip: [],
+    // Everything but Alfajores and Mumbai
+    chainsToSkip: [
```

Comment:
I think with this, you actually want to enlargen the `fullCycleTime`, otherwise I think we are sending quite frequently still?

Reply:
Doesn't this work out to 1 message per hour?